# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/bindings/Bindings.conf
# Commit: 92b87b0f776c
# Full Hash: 92b87b0f776c3049499038d380190351e2371324
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-04 03:56:33
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part2: Switch Localization class to use localization-ffi. r=emilio,nika
#   
#   Depends on D104788
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104789
# ==============================================================================

diff -r 878b052ebeaa -r 92b87b0f776c dom/bindings/Bindings.conf
--- a/dom/bindings/Bindings.conf	Tue Aug 03 16:25:10 2021 +0000
+++ b/dom/bindings/Bindings.conf	Tue Aug 03 16:25:10 2021 +0000
@@ -465,7 +465,6 @@
 },
 
 'Localization': {
-    'implicitJSContext': [ 'formatValue', 'formatValues', 'formatMessages', 'formatValueSync', 'formatValuesSync', 'formatMessagesSync' ],
     'nativeType': 'mozilla::intl::Localization',
 },
 